docs/whatsnew/5.2.rst

```diff
@@ -116,6 +116,9 @@ Performance Improvements
 To help speed up coordinate transformations, several performance improvements
 were implemented, mainly concerning validity checks in ``Angle`` and
 how ``FrameAttributes``  are accessed.
+Performance improvements will vary between different tasks, e.g. the speedup of
+transforming ``AltAz`` coordinates to ``ICRS`` is 5-10 % depending on the number
```

Comment:
Differently from the change log the What's New page can contain hyperlinks, so there is no need to hold back:
```suggestion
transforming :class:`~astropy.coordinates.AltAz` coordinates to :class:`~astropy.coordinates.ICRS` is 5-10 % depending on the number
```